manual/spotify/spotify.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>spotify</id>
+    <version>1.0.49.125</version>
+    <title>Spotify</title>
+    <authors>Spotify</authors>
+    <projectUrl>http://spotify.com</projectUrl>
+    <owners>chocolatey ferventcoder</owners>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/spotify</packageSourceUrl>
+    <licenseUrl>https://www.spotify.com/us/legal/end-user-agreement/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/spotify.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+      ## Features
+      - Millions of tracks, any time you like.
+      - Search for it in Spotify, then play it. 
+      - Help yourself to whatever you want, whenever you want it.
+    </description>
+    <summary>Spotify is a new way to listen to music</summary>
+    <copyright>Copyright(c) 2017, Spotify Ltd</copyright>
```

Comment:
this should probably be changed to `Copyright © 2017 Spotify AB`, as shown in the software and on the webpage (possibly without the Copyright part)